web/viewer.css

```diff
@@ -304,6 +304,10 @@ body {
   inset: 32px 0 0;
   outline: none;
 }
+#viewerContainer.noUserSelect {
+  user-select: none;
+}
+
```

Comment:
To not unnecessarily break the viewer-components, please add this new CSS-rule on the `.pdfViewer` instead and thus placed in https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.css